sources/api/middleware/authStrategies/AuthStrategyFacebook.ts

```diff
@@ -5,7 +5,7 @@ import * as RequestPromise from "request-promise";
 
 import { User } from "../../../model/entity/User";
 import { UserView } from "../../views/UserView";
-import { FacebookProviderPayload } from "../../../utils/types/ProviderPayload";
+import { emptyFacebookPayloadToken } from "../../../utils/emptyProviderPayload";
```

Comment:
Make me realized how random is the `utils` folder content 😕 (you don't need to take any action here)

Reply:
Yes we should create to issue to rearrange it a bit